Containerfile.mass-rebuild

```diff
@@ -1,6 +1,12 @@
-FROM registry.fedoraproject.org/fedora:41
-
-ENV LLVM_SYSROOT=/opt/llvm \
+ARG arch=x86_64
```

Comment:
I don't understand what will happen when building for other architectures.
Is this assignment ignored?

Reply:
That's just the default for when someone doesn't specify they arch option, but I've dropped this now based on your suggestion below.